theano/sandbox/rng_mrg.py

```diff
@@ -1151,7 +887,7 @@ def updates(self):
         # TODO : need description for method and return
         return list(self.state_updates)
 
-    def __init__(self, seed=12345, use_cuda=None):
+    def __init__(self, seed=12345):
```

Comment:
I would prefer to get rid of it.  We will get an error anyway if something tries to use it.